mantle/cmd/kola/qemuexec.go

```diff
@@ -125,6 +125,39 @@ func parseBindOpt(s string) (string, string, error) {
 	return parts[0], parts[1], nil
 }
 
+func buildDiskFromOptions() *platform.Disk {
```

Comment:
Probably doc comment worthy like
```suggestion
// buildDiskFromOptions generates a disk image template using the process-global defaults that were parsed from command line arguments.
func buildDiskFromOptions() *platform.Disk {
```